src/NUnitTestAdapter/NUnitEventListener.cs

```diff
@@ -163,7 +163,16 @@ public void TestFinished(NUnitTestEventTestCase resultNode)
 
             var result = _testConverter.GetVsTestResults(resultNode, outputNodes ?? EmptyNodes);
             if (_settings.ConsoleOut == 1 && !string.IsNullOrEmpty(result.ConsoleOutput))
+            {
                 _recorder.SendMessage(TestMessageLevel.Informational, result.ConsoleOutput);
+            }
+            if (_settings.ConsoleOut == 1 && !string.IsNullOrEmpty(resultNode.ReasonMessage))
+            {
+                var msg = resultNode.ReasonMessage;
+                var msg2 = $"{resultNode.Name}: {msg}";
+                _recorder.SendMessage(TestMessageLevel.Informational, msg2);
```

Comment:
Consider avoiding variable names suffixed with '2' (can we inline both the variables?)

Reply:
Done